components/brave_sync/brave_sync_service_unittest.cc

```diff
@@ -1259,3 +1275,277 @@ TEST_F(BraveSyncServiceTest, DeviceIdV2MigrationDupDeviceId) {
       base::BindOnce(&OnGetRecordsStub);
   sync_service()->OnPollSyncCycle(std::move(on_get_records), &we);
 }
+
+TEST_F(BraveSyncServiceTest, IsOtherBookmarksFolder) {
+  AsMutable(model()->other_node())
+    ->SetMetaInfo("object_id", GenerateObjectIdForOtherNode(std::string()));
+  auto record_matches_id = SimpleFolderSyncRecord(
+      SyncRecord::Action::A_CREATE, GenerateObjectIdForOtherNode(std::string()),
+      "", "", "", "", false, "");
+  EXPECT_TRUE(sync_service()->IsOtherBookmarksFolder(record_matches_id.get()));
+
+  auto record_matches_traits = SimpleFolderSyncRecord(
+      SyncRecord::Action::A_CREATE, GenerateObjectIdForOtherNode("123"),
+      GetOtherNodeName(), kOtherNodeOrder, "", "", false, GetOtherNodeName());
+  EXPECT_TRUE(sync_service()
+                ->IsOtherBookmarksFolder(record_matches_traits.get()));
+
+  auto record_not_match1 = SimpleFolderSyncRecord(
+      SyncRecord::Action::A_CREATE, GenerateObjectIdForOtherNode("123"),
+      GetOtherNodeName(), "", "", "", false, GetOtherNodeName());
+  EXPECT_FALSE(sync_service()
+                ->IsOtherBookmarksFolder(record_not_match1.get()));
+
+  auto record_not_match2 = SimpleFolderSyncRecord(
+      SyncRecord::Action::A_CREATE, GenerateObjectIdForOtherNode("123"),
+      "What Bookmarks", kOtherNodeOrder, "", "", false, GetOtherNodeName());
```

Comment:
Maybe for us it would be enough to rely only on dedicated order for `Other bookmarks` and to avoid use of title because title depends of device locale?
I sometimes switch device language on my phone just for fun.

Reply:
Changing locale on mobile device doesn't matter, it won't change the name of the normal folder "Other Bookmarks" but if we have two desktops with different locales, we might have a problem.
One of them couldn't identify incoming "Other Bookmarks". I'm down for checking specific order and id but @bridiver insisted we need to check title.

Reply:
I understand